projectroles/project_tags.py

```diff
@@ -1,59 +1,24 @@
 """Functions for project tagging/starring in the projectroles app"""
 # NOTE: This can be expanded to include other types of tags later on
 
-from projectroles.models import ProjectUserTag, PROJECT_TAG_STARRED
+from projectroles.app_settings import AppSettingAPI
```

Comment:
The whole `project_tags` module should be removed, as we will no longer have the concept of a project tag. Project starring/unstarring should happen within the relevant Ajax view.